### DISCUSSION TITLE
p-table:  group rows by multiple fields

### DISCUSSION THREAD

**Discussion by DanielGarashi** [primefaces/community#1275] [PrimeNG] [answered]
Hi, there is any way to group rows by more than one field?
for example:
interface Person {
  id: string;
  name: string;
  age: string
}

group by name and age.
rowGroupMode="rowspan" 
groupRowsBy="[ 'name', 'age' ]"

group all the rows that people with the same name and the same age (**name and age are the key**)

*According the documentation(primeNg version 15) it possible:
groupRowsBy | string\|array | null | One or more field names to use in row grouping.

you can give me an example of how to pass an array of fields?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It should be a documentation mistake, table doesn't support row grouping of more than 1 field yet.
